docs/document/docs/components/desktop/table.md

```diff
@@ -6,5 +6,126 @@ original_id: table
 ---
 
 ## Overview
+The Table component allows the user to display an input table on the kintone’s edit screen.
 
-Table English language
+<div class="sample-container" id="table">
+  <div id="sample-container__components"></div>
+</div>
+<script src="/js/samples/desktop/table.js"></script>
+
+---
+
+## Specification
+
+### Property
+
+Here is a list of properties that can be used for modifying the component:
+
+#### Table
+
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| data | Array<object> | []  | Row data of table | Throw an error if the data is not an array object `'data' property is not array.` |
```

Comment:
You have to modify from: Array<object> to
Array\<object\>

Reply:
I updated it.